lib/JIRA/REST.pm

```diff
@@ -342,9 +342,10 @@ sub attach_files {
     my $rest = $self->{rest};
 
     # FIXME: How to attach all files at once?
+    my $url = $rest->getHost . $self->{ api } .  "/issue/$issueIdOrKey/attachments";
```

Comment:
Hmm... $self->{api} may not be pointing to JIRA's Core REST API. It may be pointing to the JIRA Software or the JIRA ServiceDesk REST APIs.

Since attach_files() is a utility function I think we can use the latest Core API (i.e., "/rest/api/latest") directly, no?